addons/website_blog/views/website_blog_components.xml

```diff
@@ -64,11 +68,15 @@
                        data-bs-toggle="dropdown"
                        aria-expanded="false">
                         <span t-if="blog" class="text-truncate" t-out="blog.name"/>
-                        <span t-else="" class="text-truncate">All</span>
+                        <span t-else="" class="text-truncate">
+                            <t t-call="website_blog.first_breadcrumb"/>
+                        </span>
                     </a>
                     <ul class="dropdown-menu dropdown-menu-end">
                         <li>
-                            <a href="/blog" t-attf-class="dropdown-item #{(not blog) and 'active'}" title="All Blogs">All</a>
+                            <a href="/blog" t-attf-class="dropdown-item #{(not blog) and 'active'}" title="All Blogs">
```

Comment:
After editing the text, the old title is still displayed.
![image](https://github.com/user-attachments/assets/835ff921-a20d-4d8e-b94d-7649af6d97c2)